django_react_proj/users-fe/src/components/UsersListRow/UsersListRow.js

```diff
@@ -27,7 +27,9 @@ const UsersListRow = ({ user, resetState }) => {
     <tr className={taskFlag ? styles.ButtonTrue : styles.ButtonFalse} key={user.id}>
       <td>{user.name}</td>
       <td className={styles.Svg}>{user.password}</td>
-      <td>{user.deadline}</td>
+      <td>
+        {user.deadline} - {user.deadlinev2}
```

Comment:
Those lines are already in this PR https://github.com/nokia-wroclaw/innovativeproject-dailyplanner/pull/25